From 26d8d5e19f7e4d23060f0b7ab4d0f876ffcfd480 Mon Sep 17 00:00:00 2001
From: Lans Zhang <jia.zhang@windriver.com>
Date: Tue, 13 Jan 2015 16:30:58 +0800
Subject: [PATCH 30/31] Disable F00F bug workaround.

Quark SoC doesn't have this bug any more, thus remove Quark SoC from the
dependency.

Signed-off-by: Lans Zhang <jia.zhang@windriver.com>
---
 arch/x86/Kconfig.cpu |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/x86/Kconfig.cpu b/arch/x86/Kconfig.cpu
index 14657ae..dc16898 100644
--- a/arch/x86/Kconfig.cpu
+++ b/arch/x86/Kconfig.cpu
@@ -319,7 +319,7 @@ config X86_PPRO_FENCE
 
 config X86_F00F_BUG
 	def_bool y
-	depends on (M586MMX || M586TSC || M586 || M486) && !PAX_KERNEXEC
+	depends on (M586MMX || M586TSC && !INTEL_QUARK_X1000_SOC || M586 || M486) && !PAX_KERNEXEC
 
 config X86_INVD_BUG
 	def_bool y
-- 
1.7.5.4

